src/common/utilities.hpp

```diff
@@ -133,6 +146,7 @@ namespace rathena {
 				vector.clear();
 			else
 				vector.erase(vector.begin() + index);
+      vector.shrink_to_fit();
```

Comment:
I don't think this is a good idea. You are reallocating the memory, every time you erase something. That could be heavy...